shub/image/build.py

```diff
@@ -39,18 +39,21 @@
               help="stream build logs to console")
 @click.option("-V", "--version", help="release version")
 @click.option("-S", "--skip-tests", help="skip testing image", is_flag=True)
-def cli(target, debug, verbose, version, skip_tests):
-    build_cmd(target, version, skip_tests)
+@click.option("-f", "--file", "filename",
+              help="Name of the Dockerfile (Default is 'PATH/Dockerfile')")
```

Comment:
We can use `default='Dockerfile'` in this option definition (it'll also show up nicely in the help string then).